charts/px-central/values.yaml

```diff
@@ -204,8 +204,7 @@ securityContext:
 tolerations: []
 
 images:
-  pullSecrets:
-    - docregistry-secret
+  pullSecrets: []
```

Comment:
When i was discussing with @diptiranjanpx on the original issue, he was saying empty string here was causing issue on MKE. Can we check once on MKE?

Reply:
@prashanthpx , It should be an empty array instead of empty string , will check it in MKE

Reply:
Tested in MKE, the deployment was successful , have attached in the PR description
@prashanthpx